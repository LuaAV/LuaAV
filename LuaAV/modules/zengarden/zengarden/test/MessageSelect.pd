#N canvas 1070 406 450 300 10;
#X obj 94 -67 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X obj 133 -69 loadbang;
#X msg 144 9 64;
#X msg 94 10 -32.2;
#X obj 94 -33 t b b b;
#X obj 289 107 print nomatch;
#X obj 124 108 print second;
#X obj 45 108 print first;
#X obj 210 107 print third;
#X obj 93 67 select 64 -32.2 0;
#X msg 180 10 0;
#X connect 0 0 4 0;
#X connect 1 0 4 0;
#X connect 2 0 9 0;
#X connect 3 0 9 0;
#X connect 4 0 3 0;
#X connect 4 1 2 0;
#X connect 4 2 10 0;
#X connect 9 0 7 0;
#X connect 9 1 6 0;
#X connect 9 2 8 0;
#X connect 9 3 5 0;
#X connect 10 0 9 0;
