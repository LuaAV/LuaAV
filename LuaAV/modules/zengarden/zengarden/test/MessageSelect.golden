[@ 0.000ms] third: bang
[@ 0.000ms] first: bang
[@ 0.000ms] second: bang